packages/web3-eth/types/index.d.ts

```diff
@@ -439,3 +446,10 @@ export interface StorageProof {
     value: string;
     proof: string[];
 }
+
+export interface FeeHistoryResult {
+    oldestBlock: string;
```

Comment:
This specifies `oldestBlock` is returned as a string but the tests are showing its returned as a number here: 
https://github.com/ChainSafe/web3.js/pull/4191/files#diff-99b1655ae7b89f474d7f663a07e45ccaa75c9270bb15bf852f1f0bdf2164f711R23

Reply:
You're right, I just blindly followed the RPC docs, thank you